src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -550,6 +561,55 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			// is this request the full path to a real file?
+			if (File.Exists(localPath) && Path.IsPathRooted(localPath))
+			{
+
+				// currently this only applies to languageDisplay.css, settingsCollectionStyles.css, and customCollectionStyles.css
+				var cssFile = Path.GetFileName(localPath);
+				if ((cssFile == "languageDisplay.css") || (cssFile == "settingsCollectionStyles.css") || (cssFile == "customCollectionStyles.css"))
+				{
+					info.ContentType = "text/css";
+					info.ReplyWithFileContent(localPath);
+					return true;
+				}
+			}
+
+			// if not a full path, try to find the correct file
+			var fileName = localPath;
```

Comment:
Compare this with Steve's recent PR for handling # in file paths. Do we have trouble finding any of our css files if the book folder name includes #?